lang/cs/Org.Apache.REEF.Wake/Remote/AuxillaryStreamingFunctions.cs

```diff
@@ -0,0 +1,275 @@
+﻿using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Org.Apache.REEF.Wake.Remote
+{
+    /// <summary>
+    /// Static class containing functions to read and write some basic types from/to stream
+    /// </summary>
+    public static class AuxillaryStreamingFunctions
```

Comment:
I did some digging, and it looks like the [BinaryReader](https://msdn.microsoft.com/en-us/library/system.io.binaryreader%28v=vs.110%29.aspx) and [BinaryWriter](https://msdn.microsoft.com/en-us/library/system.io.binarywriter%28v=vs.110%29.aspx) classes in C# already provide this. Maybe we can just use those?